### ISSUE TITLE
[bug] will check the not opened profiles after restart

### ISSUE THREAD

**Issue by tjx666** [streetsidesoftware/vscode-spell-checker#3400]
reproduce:

1. quit the vscode by CMD + Q
2. starup vscode
3. you will the many suggestion reported by vscode-cspell-checker

<img width="1449" alt="image" src="https://github.com/streetsidesoftware/vscode-spell-checker/assets/41773861/a8dcc183-c2b6-4377-8b93-39eb3cc0593c">

## Note

If you reopen vscode by command `Developer: Reload Window`, no this issue. This issue only happen when `re-startup`, not `re-open`

```
Version: 1.90.2 (Universal)
Commit: 5437499feb04f7a586f677b155b039bc2b3669eb
Date: 2024-06-18T22:37:41.291Z
Electron: 29.4.0
ElectronBuildId: 9728852
Chromium: 122.0.6261.156
Node.js: 20.9.0
V8: 12.2.281.27-electron.0
OS: Darwin arm64 23.5.0
cspell-extension: v3.0.1
```

**Comment by Jason3S** [comment]
@tjx666,

## tl;dr

I think this is caused by a change in how VSCode opens configuration files. 

I'll see about changing it to only show files visual in the editor. 

## Detail

In the past VS Code opened configuration files in the background. I think it now uses the same system to load the configuration as the editors. As a result, all extension that are listening get notified that a file was opened. Since the file was opened, the spell checker checks it.
